.github/workflows/check-links.yml

```diff
@@ -31,6 +31,15 @@ jobs:
         # 429: Too many requests
         args: >
           --accept 429
+          --exclude "^http://www.seis.sc.edu/software/distaz/distaz.c/n$"
+          --exclude "^http://www.seis.sc.edu/software/distaz/distaz.c/n/n#$"
+          --exclude "^https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb/n$"
+          --exclude "^https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB/n#$"
```

Comment:
lychee sometimes doing bad with parsing URLs. Perhaps we should exclude all URLs ending with `/n` and `/n#`?

Reply:
`"*/n$"`, `"*/n#$"`?

Reply:
done in d234b2c.

Reply:
It may work or not. You can enable the workflow in this PR to check how it works.

Reply:
Good. Works (#214)

Reply:
No, it doesn't work.